qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -149,13 +153,15 @@ def getNodeItemForNode(self, node, highlight_level):
                              label=name,
                              shape=node.attr.get('shape', 'ellipse'),
                              color=color,
-                             tooltip=node.attr.get('tooltip')
-                             #parent=None,
+                             tooltip=node.attr.get('tooltip'),
+                             parent=scene.activePanel() if scene is not None else None
                              #label_pos=None
                              )
+        if scene is not None:
+            scene.addItem(node_item)
         return node_item
 
-    def addEdgeItem(self, edge, nodes, edges, highlight_level, same_label_siblings=False):
+    def addEdgeItem(self, edge, nodes, edges, highlight_level, scene=None, same_label_siblings=False):
```

Comment:
Please add the new keyword argument `scene` at the end of the signature. That will avoid breaking any code using the current signature without specifying the keyword explicitly.

Same below for `dotcode_to_qt_items`.